MMM-pages.js

```diff
@@ -178,14 +176,14 @@ Module.register('MMM-pages', {
    * elements.
    */
   updatePages: function () {
-    // Update iff there's at least one page.
+    // Update if there's at least one page.
     if (this.config.modules.length !== 0) {
       this.animatePageChange();
       if (!this.rotationPaused) {
         this.resetTimerWithDelay(0);
       }
       this.sendNotification('NEW_PAGE', this.curPage);
-    } else { Log.error("[Pages]: Pages aren't properly defined!"); }
+    } else { Log.error('[MMM-pages] Pages aren\'t properly defined!'); }
```

Comment:
followup: I feel like adding an escape just to use single quotes feels a little extra.

If you really want to stay consistent with single quotes I would rather just change the phrasing to say `are not` to avoid the escape.
```suggestion
    } else { Log.error('[MMM-pages] Pages are not properly defined!'); }
```

Reply:
I will revise ESLint in the next step and there will also be a quotes rule. Since you seem to favour single quotes, I didn't want to change everything to double quotes. The rewording is a fantastic idea :+1: